add-chain/main.go

```diff
@@ -158,6 +161,26 @@ func entrypoint(ctx *cli.Context) error {
 		return fmt.Errorf("error generating chain config .yaml file: %w", err)
 	}
 
+	// Ensure rollup config generated by op-node is equivalent to
+	// pre-existing one used to create superchain config files
+	opNodeRollupConfig, err := rollup.LoadOPStackRollupConfig(rollupConfig.ChainID)
+	if err != nil {
+		return fmt.Errorf("failed when using op-node to generate rollup config: %w", err)
+	}
+	fmt.Printf("Attempting to read from %s\n", rollupConfigPath)
+	file, err := os.ReadFile(rollupConfigPath)
+	if err != nil {
+		return fmt.Errorf("failed to read from local rollup.json config file: %w", err)
+	}
+	var localRollupConfig *rollup.Config
+	if err = json.Unmarshal(file, &localRollupConfig); err != nil {
+		return fmt.Errorf("failed to unmarshal local rollup.json into rollup.Config struct: %w", err)
+	}
+
+	if !reflect.DeepEqual(opNodeRollupConfig, localRollupConfig) {
```

Comment:
We could use https://pkg.go.dev/github.com/google/go-cmp/cmp#Diff to get some more useful output in the case where the configs are not equal.